server.js

```diff
@@ -42,9 +42,56 @@ server.use('/dev', proxy(DEVSERVER))
 server.listen(PORT, () => console.log(`listening on port ${PORT}`))
 
 
+const read_active_filter = ({on_filter}) => {
+    // Step 1: See if there is an active context
+    const tw_context = readline.createInterface({
+        input: spawn_tw(['_get', 'rc.context']).stdout
+    })
+    let context = ''
+    tw_context.on('line', (line) => {
+        if (!context && line) context = line
+    })
+    tw_context.on('close', (line) => {
```

Comment:
According to https://nodejs.org/api/readline.html#readline_event_close the `close` callback does not receive a `line` argument: "The listener function is called without passing any arguments."

Reply:
Oops :-D, I am pretty sure I thought "hum, pretty sure that's not like that" then forgot 20 seconds afterwards and copy-pasta happened.